examples/howto/blackbox_external_likelihood_numpy.myst.md

```diff
@@ -409,23 +536,31 @@ az.plot_pair(idata_grad, **pair_kwargs, ax=ax)
 # Plot pure pymc+nuts fit (green)
 pair_kwargs["kde_kwargs"]["contour_kwargs"]["colors"] = "C2"
 pair_kwargs["marginal_kwargs"]["color"] = "C2"
-az.plot_pair(idata, **pair_kwargs, ax=ax);
+az.plot_pair(idata_pure, **pair_kwargs, ax=ax);
 ```
 
-We can now check that the gradient Op works as expected. First, just create and call the `LogLikeGrad` class, which should return the gradient directly (note that we have to create a {ref}`PyTensor function <pytensor:creating_an_op>` to convert the output of the Op to an array). Secondly, we call the gradient from `LogLikeWithGrad` by using the {func}`grad` function. Finally, we will check the gradient returned by the PyMC model for a Normal distribution, which should be the same as the log-likelihood function we defined. In all cases we evaluate the gradients at the true values of the model function (the straight line) that was created.
+## Using a Potential instead of CustomDist
+
++++
+
+In this section we show how a {class}`pymc.Potential` can be used to implement a black-box likelihood a bit more directly than when using a a {class}`pymc.CustomDist`. 
```

Comment:
```suggestion
In this section we show how a {func}`pymc.Potential` can be used to implement a black-box likelihood a bit more directly than when using a a {class}`pymc.CustomDist`. 
```

Potential is actually a function, so {class} wouldn't work for the reference. Similarly to pymc.Slice though, it won't work yet either until we revert and fix pymc api docs, which currently document it at `pymc.model.core.Potential`. Same goes for all other potential references, they should be `` {func}`pymc.Potential` ``

Reply:
Fixed, sneaky PyMC capitalized functions